pixi/mocks/pageExperienceCheck/reportDataErrors.json

```diff
@@ -40,7 +40,7 @@
 			}
 		},
 		"labData": {
-			"isAllFast": false,
+			"isAllFast": true,
```

Comment:
Can you instead update the scores such that the `category` properties do not have to change?

Reply:
Good point, thanks!